static/js/constants.ts

```diff
@@ -207,3 +207,7 @@ export enum WebsiteStarterStatus {
   Active = "active",
   Inactive = "inactive"
 }
+
+export enum Permissions {
+  canCreateSites = "canCreateSites"
```

Comment:
```suggestion
  canAddWebsite = "canAddWebsite"
```
I think `canAddWebsite` instead of `canCreateSites` would be more suitable and similar to the django permission.
or maybe we can make permissions something like this: 
(just a thought)
```
permissions = {
    website: {
        view: "value", 
        add: "value", 
        change: "value", 
        delete: "value" 
    }
}
```

Reply:
Good Catch, I have updated the code as required and as for your thought for creating a permission object. it depends on the scope.